src/app/screens/Keysend/index.tsx

```diff
@@ -119,8 +119,7 @@ function Keysend() {
                   <DualCurrencyField
                     id="amount"
                     label={t("amount.label")}
-                    min={+0 / 1000}
-                    max={+1000000 / 1000}
+                    min={0}
```

Comment:
Shouldn't the min be `1`? 

```suggestion
                    min={1}
```